openedx/adg/lms/applications/models.py

```diff
@@ -143,7 +143,7 @@ class Education(UserStartAndEndDates):
 
     name_of_school = models.CharField(verbose_name=_('Name of School / University'), max_length=255, )
     degree = models.CharField(verbose_name=_('Degree Received'), choices=DEGREE_TYPES, max_length=2, )
-    ares_of_study = models.CharField(verbose_name=_('Area of Study'), max_length=255, blank=True)
+    ares_of_study = models.CharField(verbose_name=_('Area of Study'), max_length=255, )
```

Comment:
I think we should update it, and please make sure we don't have any usage of this field in code. and also post it in group so that any other person is using this field in any of in-progress ticket he knows this.